OpenKeychain/src/main/java/org/sufficientlysecure/keychain/pgp/PgpKeyOperation.java

```diff
@@ -356,10 +361,16 @@ public PgpEditKeyResult createSecretKeyRing(SaveKeyringParcel saveParcel) {
      *
      */
     public PgpEditKeyResult modifySecretKeyRing(CanonicalizedSecretKeyRing wsKR,
+                                                CryptoInputParcel cryptoInput,
+                                                SaveKeyringParcel saveParcel) {
+        return modifySecretKeyRing(wsKR, cryptoInput, saveParcel, new OperationLog());
+    }
+
+    public PgpEditKeyResult modifySecretKeyRing(CanonicalizedSecretKeyRing wsKR,
             CryptoInputParcel cryptoInput,
-            SaveKeyringParcel saveParcel) {
+            SaveKeyringParcel saveParcel,
+            OperationLog log) {
```

Comment:
log should always be passed together with indent, if it's supposed to be logged as part of another action